pkgs/by-name/re/renovate/package.nix

```
@@ -20,13 +21,13 @@ let
 in
 stdenv'.mkDerivation (finalAttrs: {
   pname = "renovate";
-  version = "38.105.2";
+  version = "39.42.4";
 
   src = fetchFromGitHub {
     owner = "renovatebot";
     repo = "renovate";
     rev = "refs/tags/${finalAttrs.version}";
```

```suggestion
    tag = finalAttrs.version;
```
You can now use the `tag` argument.

---

TIL, thats cool, thanks!